dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageErrorCause.java

```diff
@@ -22,6 +22,8 @@
 @AutoValue
 // TODO "cause" is unclear. Is it the target or the source? I don't know.
 // Rename things to remove the word "cause" and use more specific terminology.
+// why do we need this class at all? It's just a wrapper around a SymbolNotResolvable.
```

Comment:
I think SNR has the bad target symbol and the reason it's bad (and whether the source of the error is reachable [?]). This has just the target symbol.